src/redux/reducers/user.js

```diff
@@ -1,17 +1,19 @@
-import {SET_USER_AUTH_STATE} from "../actions/auth";
+import { SET_USER_AUTH_STATE, SIGN_OUT_USER } from "../actions/auth";
 
 const initialState = {
-    isLoggedIn: false,
-    token: null
+  isLoggedIn: false,
+  token: null,
 };
 
-const user = (state = initialState , action) => {
-    switch (action.type) {
-        case SET_USER_AUTH_STATE:
-            return { ...state, isLoggedIn: action.isLoggedIn, token: action.token};
-        default:
-            return state;
-    }
+const user = (state = initialState, action) => {
+  switch (action.type) {
+    case SET_USER_AUTH_STATE:
```

Comment:
We should Rename this to `LOGIN_USER` if we are going to be using `SIGN_OUT` as separate action.

Reply:
Have you done this or should I go ahead and do it?